libs/ui-components/src/components/form/validations.ts

```diff
@@ -653,8 +653,8 @@ export const systemdUnitListValidationSchema = (t: TFunction) =>
       }),
     )
     .test('invalid patterns', (systemdUnits: SystemdUnitFormValue[] | undefined, testContext) => {
-      // TODO analyze https://github.com/systemd/systemd/blob/9cebda59e818cdb89dc1e53ab5bb51b91b3dc3ff/src/basic/unit-name.c#L42
-      // and adjust the regular expression and / or the validation to accommodate for it
+      // Supports templated SystemD services with extended regex for all allowed unit file formats and glob searches
+      // Pattern supports: [0-9a-zA-Z:-_.\\\[\]!-*?]+(@[0-9a-zA-Z:-_.\\\[\]!-*?]+)?(\.[a-zA-Z\[\]!-*?]+)?
```

Comment:
```suggestion
      // See SYSTEMD_PATTERNS_REGEXP
```
as otherwise the comment may easily become outdated

Or maybe move the comment on top of the RegExp so anyone checking it understands what type of systemd services it allows.